backend/lib/endpoints/posts.js

```diff
@@ -396,15 +396,16 @@ async function routes(app) {
         // user shouldn't see posts reported by them, even if public.
         const didReport = post.reportedBy
           ? post.reportedBy.find(
-              (r) => r.id.toString() === actor._id.toString(),
-            )
+            (r) => r.id.toString() === actor._id.toString(),
+          )
           : false;
         if (didReport) throw app.httpErrors.notFound();
       } else if (!actor) {
         // none logged in user shouldn't see removed posts on post page
         if (post.status === "removed") throw app.httpErrors.notFound();
       }
-      
+      post.views += 1;
```

Comment:
maybe change to use `findByIdAndUpdate` with `$inc` clause, which will be atomic based on [this article](https://stackoverflow.com/questions/36438390/in-mongoose-is-model-save-atomic-for-increment-or-should-i-always-use-inc)  
also add error handling like [this line ](https://github.com/FightPandemics/FightPandemics/blob/50edf4a5b84f16dbbffb864e85772715275023b3/backend/lib/endpoints/reports.js#L203)